trio/_tests/test_exports.py

```diff
@@ -503,7 +505,7 @@ def test_classes_are_final():
                 continue
             # These are classes that are conceptually abstract, but
             # inspect.isabstract returns False for boring reasons.
-            if class_ in {trio.abc.Instrument, trio.socket.SocketType}:
+            if class_ in (trio.abc.Instrument, trio.socket.SocketType):
```

Comment:
I thought `set.__contains__` was slightly more efficient than tuple's, but I might be wrong

Reply:
It would be, but for sets mypy evaluates the type here as something like `set[type[object]]`, whereas for tuples it can be `tuple[type[trio.abc.Instrument], type[trio.socket.SocketType]]`, so it doesn't error. Might be even better to just unroll the two compares.

Reply:
If it's helping the type checker understand I think it's fine then